docker/run.sh

```diff
@@ -113,6 +115,7 @@ sudo docker run --privileged --net=host -it $NVIDIA_FLAGS \
        -v ${REPOSITORY_FOLDER_PATH}/.build:$WORKSPACE_ROOT_CONTAINER/build:rw \
        -v ${REPOSITORY_FOLDER_PATH}/.install:$WORKSPACE_ROOT_CONTAINER/install:rw \
        -v $SSH_PATH:$SSH_PATH \
+       -u 1000:1000 \
```

Comment:
Is it ok to use an arbitrary uid and gid?

Reply:
It's not entirely arbitrary, that's the uid and gid that was used with my user on Ubuntu22, so I tried to replicate that behavior as much as I could on Ubuntu24. I think we should be fine for now.